src/lighteval/tasks/requests.py

```diff
@@ -178,7 +178,7 @@ class Doc:
 
     # For few-shot
     instruction: Optional[str] = ""
-    target_for_fewshot_sorting: Optional[str] = None  # will probably have to be removed in the future
+    fewshot_sorting_class: Optional[str] = None  # class to use to select balanced few-shot samples
```

Comment:
what does `class` means here ?

Reply:
Remember how we do the balanced few shot sorting in the normal setup? 
We sort the samples by their gold label, and select the same number of each (so that we get an even number of A, B, C, D predictions in the few shot examples, to avoid bias towards a given label). Here, the gold is the "class" we use for few shot sorting.
In some other cases (if you look at the original PR this references), you need to use something else as the class (for example, a sample attribute).